src/otk/transform.py

```diff
@@ -153,6 +153,9 @@ def process_defines(ctx: Context, state: State, tree: Any) -> None:
     if tree is None:
         log.warning("empty otk.define in %s", state.path)
         return
+    if tree == {}:
```

Comment:
Do we want to do this for `[]` as well?

Reply:
I think we should.

Reply:
But wait, here `tree` is always a dictionary.  I think `[]` might work as is right now.